internal/api/keppel/accounts_test.go

```diff
@@ -1933,138 +1870,32 @@ func TestDeleteAccount(t *testing.T) {
 	}.Check(t, h)
 
 	// failure case: account not in maintenance
-	_, err = s.DB.Exec(`UPDATE accounts SET in_maintenance = FALSE`)
-	if err != nil {
-		t.Fatal(err.Error())
-	}
-	assert.HTTPRequest{
-		Method:       "DELETE",
-		Path:         "/keppel/v1/accounts/test1",
-		Header:       map[string]string{"X-Test-Perms": "view:tenant1,change:tenant1"},
-		ExpectStatus: http.StatusConflict,
-		ExpectBody: assert.JSONObject{
-			"error": "account must be set in maintenance first",
-		},
-	}.Check(t, h)
-	_, err = s.DB.Exec(`UPDATE accounts SET in_maintenance = TRUE`)
+	_, err = s.DB.Exec(`UPDATE accounts SET is_deleting = FALSE`)
 	if err != nil {
 		t.Fatal(err.Error())
 	}
 
-	// phase 1: DELETE on account should complain about remaining manifests
+	// phase 1: DELETE on account should immeadetly mark it for deletion
```

Comment:
Please double-check for typos:
```suggestion
	// phase 1: DELETE on account should immediately mark it for deletion
```